crates/containerd-shim-wasm/src/sandbox/sys/unix/stdio.rs

```diff
@@ -0,0 +1,9 @@
+use std::io::Result;
+pub use std::os::fd::AsRawFd;
+use std::os::fd::OwnedFd;
+
+pub use libc::{STDERR_FILENO, STDIN_FILENO, STDOUT_FILENO};
+
+pub fn try_into_fd(f: impl Into<OwnedFd>) -> Result<impl AsRawFd> {
+    Ok(f.into())
```

Comment:
It looks like this function will never return Error. Why don't we remove the `try` prefix?

Reply:
On windows this operation can fail.
It returns a result here so that we can use the same interface in both unix and windows.